src/encoded/tests/test_schema_functional_characterization_experiment.py

```diff
@@ -40,6 +40,18 @@ def functional_characterization_experiment(testapp, lab, award, cell_free):
     return testapp.post_json('/functional_characterization_experiment', item).json['@graph'][0]
 
 
+@pytest.fixture
```

Comment:
@gabdank for the record, I think we are going to start moving ALL fixtures to datafixtures.py.

Reply:
(I am going to merge this anyway... but in the future let's put fixtures in datafixtures.py